extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -245,6 +245,21 @@ export const popupMessageListener = (request: Request) => {
     });
   };
 
+  const _makeAccountActive = async ({ publicKey }: { publicKey: string }) => {
```

Comment:
NIT: can you name this something like `_activatePublicKey` or something? Just want to avoid having 2 functions named really similarly. Makes it a little easier to search in the codebase

Reply:
ya good idea, changed in https://github.com/stellar/freighter/pull/679